foundations/installations/installations.md

```diff
@@ -84,7 +84,11 @@ In the **Hardware** section of the installation you want to set your **Base Memo
 
 For example, if you have 8 GB (8192 MB respectively) of RAM, you could allocate up to 4096 MB (1024 MB to 1 GB) to your VM’s operating system. If you do not know how much RAM is available to you, [please run this Google query](https://www.google.com/search?q=how+to+find+out+how+much+ram+you+have) to learn how to find this out. If the VM runs a bit slow, try allocating more memory!
 
-_(__note:__ Difficulty converting your **G**iga**B**ytes into **M**ega**B**ytes? 1 GB of RAM is equal to 1024 MB. Therefore, you can say that **8 GB = 8 x 1024 = 8192 MB.**)_
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+Difficulty converting your <strong>G</strong>iga<strong>B</strong>ytes into <strong>M</strong>ega<strong>B</strong>ytes? 1 GB of RAM is equal to 1024 MB. Therefore, you can say that <strong>8 GB = 8 x 1024 = 8192 MB.</strong>
```

Comment:
Bolding the letters doesn't make much sense here, instead we should just include the abbreviation in parenthesis. Also swapping out the strong tag for the `b` tag since semantically the equation isn't really "important", rather we just want to make it stand out visually.

```suggestion
Difficulty converting your Gigabytes (GB) into Megabytes (MB)? 1 GB of RAM is equal to 1024 MB. Therefore, you can say that <b>8 GB = 8 x 1024 = 8192 MB.</b>
```